### DISCUSSION TITLE
Override the domain URL?

### DISCUSSION THREAD

**Discussion by simekent** [pbteja1998/remix-auth-email-link#27] [Q&A] [answered]
Is there an easy way to specify the domain URL? We have some tests that use "https" on localhost but this line here hardcodes to "http":

https://github.com/pbteja1998/remix-auth-email-link/blob/main/src/index.ts#L318

We currently fix up the domain in the SendEmailFunction implementation but it's a bit of a hack.

If there aren't supported ways, allowing the domain URL to be specified or overridden (by making getDomainURL protected?) would work. Alternatively, respecting the "x-forwarded-proto" header would also work because we run behind a reverse-proxy.

**Comment by pbteja1998** [comment, marked as answer]
Done. It will now respect the `X-Forwarded-Proto` header if available.

Published in `v1.5.2`

**Reply by simekent** [reply]
Awesome, thanks for the quick turnaround!
